File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/SeatAssigning/Models/SeatAssignment.cs
using System;

namespace Registration.SeatAssigning
{
    public class SeatAssignment
    {
        public int Position { get; private set; }
        public SeatType Seat { get; private set; }
        public Attendee Attendee { get; set; }

        public SeatAssignment() { }
        public SeatAssignment(int position, SeatType seat)
        {
            Position = position;
            Seat = seat;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/seatassigning/Models/SeatAssignment.java
package com.microsoft.conference.registration.domain.seatassigning.Models;

import com.microsoft.conference.registration.domain.SeatType;

public class SeatAssignment {
    public int position;
    public SeatType seatType;
    public Attendee attendee;

    public SeatAssignment() {
    }

    public SeatAssignment(int position, SeatType seatType) {
        this.position = position;
        this.seatType = seatType;
    }
}
